### ISSUE TITLE
request for msys2/mingw64 build 

### ISSUE THREAD

**Issue by hgkamath** [PerBothner/DomTerm#84]
I notice that there is a WSL build. 
WSL, for some, its hypervisor/vm/linux subsystem may be too much. same can be said of a cygwin build.

Since release of windows-10 1903, microsoft introduced a new API conpty for linux-like pseudoterminals to phase out the older conhost. The recently released windows terminal also uses the conpty approach. 
Some tools like vim, do seem to support the new conpty. msys/cygwin seem to be attempting conpty around the 3.1.15 version.

I think it should be within reach to have a msys/mingw64 build for domterm.exe and domterm-qt.exe 
Hence the suggestion/request.

**Comment by PerBothner** [comment]
I agree this would be nice, and I would support it.  However, there are some other "big-ticket" items on the roadmap that I'm focusing on before native-Windows support.  If someone where to contribute it, it would be great.

There is an experimental ["pure-Electron" version of DomTerm](http://domterm.org/electron-nodepty.html) which might be easier to port to native Windows.  An alternative I'm also keeping an eye on is [Deno](https://deno.land/), perhaps combined with [deno-webview](https://deno.land/x/webview) - but that is too early.

**Comment by PerBothner** [comment]
I added some instructions for building and using [electron-nodepty](https://domterm.org/electron-nodepty.html), after verifying that it works on plain (non-WSL) Windows (after fixing a couple of issues).

**Comment by cebaa** [comment]
@PerBothner Just one note regarding electron-nodepty and builds in general (on Windows at least). Looks like the build requires installing Visual Studio. In many cases, installation is not even possible (e.g. no admin privileges). Having pre-built binaries is a life-saver in such cases (and also brings a bunch of other niceties). Not sure if possible and how demanding it would be to set up github actions that would build these on release.

**Comment by PerBothner** [comment]
_"Looks like the build requires installing Visual Studio."_

I assume that is because of the node-pty package.

I agree pre-built binaries would be very nice, but that's a lot of new stuff I (or someone else) would have to master. For Windows we have the issue of either getting the C-domterm working without WSL or beefing up electron-nodepty.  (Longer-term, I'm watching Deno with interest.)

I'd love to have help with these things, but for now I don't have the time.
